src/main/java/com/smartbear/readyapi/client/execution/CodegenBasedTestServerApi.java

```diff
@@ -296,8 +300,41 @@ public void setBasePath(String basePath) {
     }
 
     @Override
-    public ProjectResultReport postProject(ProjectExecutionRequest executionRequest, boolean async, HttpBasicAuth auth) throws ApiException {
+    public ProjectResultReport postSwagger(File swaggerFile, TestServerRequestExecutor.SwaggerFormat swaggerFormat,
```

Comment:
yeah - maybe that's better - perhaps we could split this into

TestServerClient client = new TestServerClient( .. )
RecipeExecutor executor = client.createExecutor(...)
ApiValidator validator = client.createValidator(...)

!?